migrace/99_migrace_notifikace_nove.sql

```diff
@@ -1,8 +1,9 @@
 -- Migrace hlídacích psů
-INSERT INTO notifikace_projekty_pes (user_id, object_id, created_at)
+INSERT INTO notifikace_projekty_pes (user_id, object_id, created_at, content_type_id)
 SELECT uzivatel, 
 (SELECT id FROM ruian_kraj WHERE ruian_kraj.id = (SELECT kraj FROM ruian_okres WHERE ruian_okres.id = (SELECT okres FROM ruian_katastr WHERE ruian_katastr.id = notifikace_projekt.katastr))) AS kraj,
-NOW()
+NOW(),
+(select id from django_content_type where model = 'ruiankraj')
```

Comment:
@jnihnat Toto v SQL migraci nemůžeme mít a nelze to tu vyřešit, protože tabulku `django_content_type` vytváří a plní až Django při buildování databáze z modelů. Je tedy třeba to spíše nějak ošetřit ve skriptu https://github.com/ARUP-CAS/aiscr-webamcr/blob/dev/migrace/copy_data.py

Reply:
@jnihnat @motyc Tam už jep připravená funkce `get_content_type`. Myslím, že by měla jít použít i na tento případ.

Reply:
@jnihnat @motyc Navrhuji toto dořešit v samostatném PR a zde udělat merge.

Reply:
Za mne ok, spíš aby to nezapadlo...